.github/scripts/generate-release-body.ts

```diff
@@ -213,10 +212,17 @@ async function main() {
   const emptyLabelPRs = prByLabels[''] || [];
 
   const printPr = (pr) => {
-    if (pr.data.labels.includes(BREAKING_CHANGES_LABEL)) {
-        return "⚠️ " + pr.data.title + " (#" + pr.data.number + ")";
+    if (pr.labels) {
+      for (const label in pr.lables) {
+        if (label == BREAKING_CHANGES_LABEL) {
+          return "⚠️ " + pr.title + " (#" + pr.number + ")";
+        }
+      }
+      return pr.title + " (#" + pr.number + ")";
+    }
+    else {
+      return pr.data.title + " (#" + pr.data.number + ")";
```

Comment:
Does this work? (Just confirmation)
I can see `.data` field was removed at other plances in this PR.